src/sync/syncTranslators.js

```diff
@@ -68,6 +68,8 @@ export const RECORD_TYPES = new SyncTranslator({
   TransactionBatch: 'trans_line',
   User: 'user',
   Unit: 'unit',
+  Pref: 'pref',
+  PrefBlob: 'pref_blob',
```

Comment:
as above I think `PrefBlob` is for explicitness? but I think it's confusing me more then helping but could be just me!

Reply:
615d1b2a42f7545707699d21d693733763b1c32e.